backend/app/management/commands/syncdb.py

```diff
@@ -209,21 +220,13 @@ def handle(self, *args, **options):
                         MapSquare.objects.filter(number=map_square_number).first()
 
                 if model_name == 'Photo':
-                    # Looks up the Photo URL of the front and back
+                    # Gets the Map Square folder and the photo number to look up the URLs
                     map_square_number = str(row.get('map_square_number', ''))
-                    srcs = row.get('srcs', '')
-                    photo_srcs = []
-                    if ',' in srcs:
-                        for src in srcs.split(','):
-                            # strip removes whitespace from beginning or end of string in case
-                            # the person who entered the data added extra spaces
-                            photo_srcs.append(src.strip())
-                    else:
-                        photo_srcs.append(srcs)
                     map_square_folder = photo_url_lookup.get(map_square_number, '')
+                    photo_number = row.get('number', '')
 
                     if map_square_folder:
-                        add_photo_srcs(model_kwargs, map_square_folder, photo_srcs)
+                        add_photo_srcs(model_kwargs, map_square_folder, str(photo_number))
```

Comment:
This is so absurdly nitpicky that I'm certainly not going to request changes for it -- but probably the better thing here is for the function to take `photo_number` as an int (since that's how it mostly will exist in the calling environment) and to do the cast in the function itself. NOT A BIG DEAL and purely a judgement call... but if this were in a situation where you'd be reusing `add_photo_srcs` a lot, it would be come inconvenient and error prone for the caller to always be responsible for that cast.